README.md

```diff
@@ -84,12 +86,25 @@ And add the following line to `.bashrc`
 eval "$(direnv hook bash)"
 ```
 
+### .envrc file
+
+Create a file in the root of the project called `.envrc` this file contains the environment variables.
+Add this lines to the file and replace the dummy values with real values.

```

Comment:
Maybe declare explicitly that if no `IP` or `PORT` are specified it will connect to `127.0.0.1:5432`? I'm not sure if it will be necessary.